clean_all.py

```diff
@@ -41,16 +42,18 @@ def is_stale(date: str) -> bool:
 
     @staticmethod
     def _iterate(
-        endpoint: DiscoveryEndpoint, key: str = "items", **kwargs
+        endpoint: DiscoveryEndpoint,
+        payload: Dict,
+        key: str = "items",
     ) -> List[Dict]:
         """
         Iterates through endpoint.list(...).execute() discovery API endpoint
 
         :param endpoint: An discovery API object for example ``self.client.instances()``
         :param key: key used to get objects from list response
-        :param **kwargs: keyword arguments passed to API list request
+        :param payload: keyword arguments passed to API list request
```

Comment:
Like the change with payload, should allow avoiding any conflicts with arguments in future services.